lib/web/apiserver.go

```diff
@@ -534,6 +534,7 @@ func (h *Handler) bindDefaultEndpoints(challengeLimiter *limiter.RateLimiter) {
 	h.GET("/scripts/:token/install-app.sh", httplib.MakeHandler(h.getAppJoinScriptHandle))
 	// web context
 	h.GET("/webapi/sites/:site/context", h.WithClusterAuth(h.getUserContext))
+	h.GET("/webapi/sites/:site/resources/check", h.WithClusterAuth(h.checkAccessToRegisteredResource))
```

Comment:
:nail_care: 

`check` seems too generic but I don't have a better suggestion
Sorry, naming is hard :bow:

Reply:
i was actually hoping someone had a better name but alas :sweat_smile: